client/stylesheet.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:ital,wght@1,200&family=Roboto&family=Varta&display=swap');
+
```

Comment:
i think were your css went wrong is that you didnt use display:flex in your classes help get the alinment correct.
also i think i whould have been good if you used 
align-items:# $ justify-content:# to help with the spacing